ostree-ext/src/container_utils.rs

```diff
@@ -64,6 +71,17 @@ pub fn is_bare_split_xattrs() -> Result<bool> {
     }
 }
 
+/// Returns true if the system appears to have been booted via ostree.
+/// This accesses global state in /run.
+pub fn is_ostree_booted() -> io::Result<bool> {
```

Comment:
```suggestion
pub fn ostree_booted() -> io::Result<bool> {
```

Just a suggestion, up to you, I feel it reads better